benchmark/cmake/modules/AddSwiftBenchmarkSuite.cmake

```diff
@@ -20,6 +20,58 @@ function(runcmd)
   set(${RUNCMD_VARIABLE} ${${RUNCMD_VARIABLE}} PARENT_SCOPE)
 endfunction(runcmd)
 
+function (swift_benchmark_library objfileout sibfileout)
```

Comment:
Can we re-name `objfileout` and `sibfileout` to something like `objfile_out` and `sibfile_out`?

Reply:
SGTM. Can I do that in a follow on commit since I already started the build? It should be trivial to know it is correct.